mono/metadata/marshal.h

```diff
@@ -269,7 +269,7 @@ gpointer
 mono_string_to_ansibstr (MonoString *string_obj);
 
 gpointer
-mono_string_to_bstr (MonoString *string_obj);
+mono_string_to_bstr (gpointer ptr, int slen);
```

Comment:
Don't change this function. It's used by marshal.c and it's going to be a mess to fix it there.